### DISCUSSION TITLE
What would be the best default shortcut to highlight a border?

### DISCUSSION THREAD

**Discussion by mipmip** [mipmip/gnome-shell-extensions-highlight-focus#23] [General]
Please reply with your personal choice.

**Comment by taq** [comment]
Super + I (hIghlight) seems to be not mapped on the current Gnome install.

**Comment by danb4r** [comment]
The default seems to be '<Super>L' not '<Super>I'. Small caps.

<Super>L conflicts with Screen Lock. I had to disable it like this:

```sh
GSETTINGS_SCHEMA_DIR=~/.local/share/gnome-shell/extensions/highlight-focus@pimsnel.com/schemas gsettings reset org.gnome.shell.extensions.highlight-focus keybinding-highlight-now
```
